data-prepper-plugins/elasticsearch/README.md

```diff
@@ -44,6 +44,9 @@ The elasticsearch sink will reserve `otel-v1-apm-service-map` as index for recor
 
 ### Custom data
```

Comment:
Lets remove this section, we are not going to support custom data officially for a while.

Reply:
Sure.